### PR TITLE
added 5-logic

### PR DESCRIPTION
**Pull request by DavidTereshchenko**
I adapted the previous homework, I will do it a little later with an asterisk)

### CODE REVIEW

File: homeworks/david.tereshchenko_DavidTereshchenko/5-logic/homework.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    let calc;
+
+    if (Number.isNaN(one) || Number.isNaN(two)) {
+        result.innerHTML = 'Enter a number';
+        return result;
+    }
+
+    if (one === 0 || two === 0) {
+        result.innerHTML = 'Enter a number';
+        return result;
```
Comment (anna-vrana-moc) [inline homeworks/david.tereshchenko_DavidTereshchenko/5-logic/homework.js:18]: Your calculator doesn't calculate 0 at all. But it should. Usually, all calculations, except for division by 0, are performed on a calculator

Comment (anna-vrana-moc) [reply homeworks/david.tereshchenko_DavidTereshchenko/5-logic/homework.js:18]: The result 'Enter a number' might be if a user typed a symbol or input field is empty

**Comment by anna-vrana-moc** [comment]
@DavidTereshchenko good fixes! Are you still planning to complete the extra task?

**Comment by DavidTereshchenko** [comment]
> @DavidTereshchenko good fixes! Are you still planning to complete the extra task?

tomorrow I will try to do it, but you can approve it, I will make a new pr if it works)

**Review by anna-vrana-moc — APPROVED** [review]

**Pull request merged**